.github/workflows/packaging.yml

```diff
@@ -0,0 +1,56 @@
+name: packaging
+
+on: [push, pull_request]
+
+jobs:
+  build_package:
+    # Skip pull request jobs when the source branch is in the same
+    # repository.
+    if: |
+      github.event_name == 'push' ||
+      github.event.pull_request.head.repo.full_name != github.repository
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - { os: 'debian', dist: 'stretch' }
+          - { os: 'debian', dist: 'buster' }
+          - { os: 'debian', dist: 'bullseye' }
+          - { os: 'el', dist: '7' }
+          - { os: 'el', dist: '8' }
+          - { os: 'fedora', dist: '30' }
+          - { os: 'fedora', dist: '31' }
+          - { os: 'fedora', dist: '32' }
+          - { os: 'fedora', dist: '33' }
+          - { os: 'fedora', dist: '34' }
+          - { os: 'opensuse-leap', dist: '15.1' }
+          - { os: 'opensuse-leap', dist: '15.2' }
+          - { os: 'ubuntu', dist: 'xenial' }
+          - { os: 'ubuntu', dist: 'bionic' }
+          - { os: 'ubuntu', dist: 'focal' }
+          - { os: 'ubuntu', dist: 'hirsute' }
+          # Skip packaging for Ubuntu Groovy due to 
+          # some troubles on Ubuntu's site
+          # - { os: 'ubuntu', dist: 'groovy' }
```

Comment:
I would not add it at all, neither uncommented, nor commented: this distro is in the EOL state: https://wiki.ubuntu.com/Releases.

Reply:
Fixed.

Reply:
I meant exactly 'don't add'. Now it is added and then removed.

Reply:
Okay. I deleted the second commit and rewrote the first.